sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -353,3 +353,23 @@ void flash_ctrl_testutils_counter_set_at_least(
                                    ARRAYSIZE(new_val)),
         "Flash write failed");
 }
+
+// At the beginning of the simulation (Verilator, VCS,etc.),
+// the content of the flash might be all-zeros, and thus,
+// the NVM counter's inital value might be 256.
+// In that case, flash_ctrl_testutils_counter_set_at_least() will not increment
+// This function can be used to initialize a NVM counter to zero by filling
+// its flash region with non-zero values.
```

Comment:
Style nitpick: documentation above a function should be a `/** ... */` style comment (and also should be in the `.h` file rather than `.c` for functions that are exposed in the header).

Reply:
Thanks, I fixed it and moved it to `.h` file.